# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ntfs-3g/glibc.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ntfs-3g_ntfsprogs-2014.2.15/src/lowntfs-3g.c.vanilla	2020-03-20 12:16:04.449677073 +0100
+++ ntfs-3g_ntfsprogs-2014.2.15/src/lowntfs-3g.c	2020-03-20 12:16:21.441677081 +0100
@@ -77,6 +77,8 @@
 #include <sys/mkdev.h>
 #endif
 
+#include <sys/sysmacros.h>
+
 #if defined(__APPLE__) || defined(__DARWIN__)
 #include <sys/dirent.h>
 #endif /* defined(__APPLE__) || defined(__DARWIN__) */
--- ntfs-3g_ntfsprogs-2014.2.15/src/ntfs-3g.c.vanilla	2020-03-20 12:16:32.017677087 +0100
+++ ntfs-3g_ntfsprogs-2014.2.15/src/ntfs-3g.c	2020-03-20 12:16:45.129677093 +0100
@@ -76,6 +76,8 @@
 #include <sys/mkdev.h>
 #endif
 
+#include <sys/sysmacros.h>
+
 #if defined(__APPLE__) || defined(__DARWIN__)
 #include <sys/dirent.h>
 #endif /* defined(__APPLE__) || defined(__DARWIN__) */
